src/sidebar/store/modules/selection.js

```diff
@@ -518,19 +544,10 @@ function focusModeHasUser(state) {
  * @return {string}
  */
 function focusModeUserPrettyName(state) {
-  const user = state.selection.focusMode.config.user;
-  if (!user) {
-    return '';
-  } else if (user.displayName) {
-    return user.displayName;
-  } else if (user.userid) {
-    return user.userid;
-  } else if (user.username) {
-    // remove once LMS no longer sends `username` in RPC
-    return user.username;
-  } else {
+  if (!focusModeConfigured(state)) {
     return '';
   }
+  return state.selection.focusMode.user.displayName;
```

Comment:
We're assuming here that there are no users of focus-mode who are _not_ providing the display name?

Reply:
Edit... nevermind. I see that `displayName` is initialized from the username or userid if there is no display name.